.github/workflows/lint-with-vale.yml

```diff
@@ -0,0 +1,20 @@
+name: lint-with-vale.yml
+on: [pull_request_target]
+
+jobs:
+  vale:
+    name: Linting with Vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: errata-ai/vale-action@reviewdog
+        with:
+          files: ./docs/modules/ROOT/pages/
+          filter_mode: added
+          reporter: github-pr-review
+          fail_on_error: true
```

Comment:
Might be a bit risky at the begining (risk of false positives due to vale->adoc bugs) ?